ignite/cmd/plugin.go

```diff
@@ -19,29 +20,46 @@ import (
 	"github.com/ignite/cli/ignite/services/plugin"
 )
 
+const (
+	igniteCmdPrefix   = "ignite "
+	flagPluginsGlobal = "global"
+)
+
 // plugins hold the list of plugin declared in the config.
 // A global variable is used so the list is accessible to the plugin commands.
 var plugins []*plugin.Plugin
 
-const (
-	igniteCmdPrefix = "ignite "
-)
-
-// LoadPlugins tries to load all the plugins found in configuration.
-// If no configuration found, it returns w/o error.
+// LoadPlugins tries to load all the plugins found in configurations.
+// If no configurations found, it returns w/o error.
 func LoadPlugins(ctx context.Context, rootCmd *cobra.Command) error {
-	cfg, err := parseLocalPlugins(rootCmd)
-	if err != nil {
-		// if binary is run where there is no plugins.yml, don't load
+	pluginsConfigs := make([]pluginsconfig.Plugin, 0)
+
+	localCfg, err := parseLocalPlugins(rootCmd)
+	if err != nil && !errors.As(err, &cosmosanalysis.ErrPathNotChain{}) {
+		fmt.Printf("error parsing local chain plugins config: %s", err.Error())
 		return nil
+
+	} else if err == nil {
+		pluginsConfigs = append(pluginsConfigs, localCfg.Plugins...)
+	}
+
+	globalCfg, err := parseGlobalPlugins()
+	if err == nil {
```

Comment:
Why not returning the error from `parseGlobalPlugins` if not nil ?

Reply:
If there is an error parsing the global config, we should probably just keep executing.  Perhaps we can log that there was a plugin error.  

I'm concerned that having an incorrect config could break things and then people will not be able to use their CLI commands.

Reply:
That's a good point, but actually this problem can arise in many other cases, like for instance if you declare a plugin and the plugin's code doesn't compile, or the repo path is not correct or private or ...

I found this becomes more and more annoying, and I plan to fix that by not canceling the `ignite` execution if there's issue with plugins. The simplest way would be to just log the error and continue, and in that case we can just do that in the caller of that `cmd.LoadPlugins`, without filtering any error from inside the function body.

Reply:
To conclude, let's keep it like I that I'll fix that later when I'll implement what I described above.